ngraph/test/eval.cpp

```diff
@@ -1810,28 +1810,6 @@ TEST(eval, topk_v1_param_dyn_k0)
     ASSERT_EQ(result1_val, expec1);
 }
 
-TEST(eval, reduce_logical_and__neg_axis)
```

Comment:
Why did you remove this test?

Reply:
The purpose of the test was to throw exception when negative axis was introduced as second input. Since negative axes are supported I removed it